### DISCUSSION TITLE
Add "References" to Item Detail sidebar

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3090] [Feature Requests]
This would be a global listing of any/all the places this item is used relationally within the database. Two styling options (grouped and not) for the linked items:

![Screen Shot 2020-08-26 at 5 48 10 PM](https://user-images.githubusercontent.com/522079/91360395-627a2600-e7c4-11ea-9cc9-967869b6a754.png)

**Comment by rijkvanzanten** [comment]
If I understand this correctly, this would show the "Item's the current item is referenced _in_" and not the "other items that are referenced in the current item" correct?

**Reply by benhaynes** [reply]
Correct. The sidebar title could be "Related Items" — with a description of: `There are 3 other items that link to this item:`

**Reply by benhaynes** [reply]
Could be called: `Links`, `References`, or `Related Items` (since relationships go both ways).

**Reply by rijkvanzanten** [reply]
I think References works best for a title (`Related Items` to me sounds like other items you've linked from this item instead of the opposite). Having that description is a good solve for that confusino

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
